WalletWasabi.Gui/Controls/WalletExplorer/ReceiveTabViewModel.cs

```diff
@@ -124,10 +124,9 @@ public ReceiveTabViewModel(WalletViewModel walletViewModel)
 				{ }
 			}, isCoinListItemSelected);
 
-			ChangeLabelCommand = ReactiveCommand.Create(() =>
-			{
-				SelectedAddress.InEditMode = true;
-			});
+#pragma warning disable IDE0053 // Use expression body for lambda expressions
```

Comment:
Why this?

Reply:
If we remove it we will get a suggestion to write the code as following:
`ChangeLabelCommand = ReactiveCommand.Create(() => SelectedAddress.InEditMode = true);`
but then we get a conversion error.